src/models/draft.js

```diff
@@ -39,6 +39,16 @@ export default class Draft extends Message {
     return super.saveRequestBody(...arguments);
   }
 
+  deleteRequestBody(params) {
+    var body = {};
+    if (params.hasOwnProperty('version')) {
```

Comment:
np: let's use ternary statement instead of this if/else statement (i.e. body.version = params.hasOwnProperty('version'))? params.version : this.version)